src/devinterp/optim/sgld.py

```diff
@@ -63,6 +63,8 @@ def __init__(
         temperature: Union[Callable, float] = 1.0,
         bounding_box_size=None,
         save_noise=False,
+        save_mala_vars=False,
+        device="cpu",
```

Comment:
is device used? i see it passed/instantiated as a class attribute but i don't see where it's called (it looks like the optimizer later infers the device type with `dw.device`)